### PR TITLE
Messing i18n key for settings

### PR DESCRIPTION
**Pull request by fhenri42**
Hello a key is messing in all the language JSON files, the key is used in this file: https://github.com/strapi/strapi/blob/dd227e2b645901ff35aa8d599ecfe8bae27850ab/packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/ContentTypeCollapse/Collapse/index.js#L235
Capture d’écran 2022-08-25 à 15 25 48
I added only EN and FR because I don't speak the other language.

Hope this help. 
Thank you for work.

### CODE REVIEW

**Comment by fhenri42** [comment]
Hello @ronronscelestes, added the change you asked. 
At the same time, I saw that you used  "ConditionsModal" in an other file, so I did the same change to keep some coherence between files. Tell me if it was a mistake, and I will roll back the change.
```js
    <ConditionsModal
        headerBreadCrumbs={[
          { id: categoryName, defaultMessage: categoryName },
          { id: subCategoryName, defaultMessage: '' }]}
          actions={formattedActions}
          isFormDisabled={isFormDisabled}
          onClosed={handleModalClose}
          onToggle={handleToggleModalIsOpen}
        />
```
path: SettingsPage/pages/Roles/EditPage/components/PluginsAndSettings/SubCategory/index.js

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/PluginsAndSettings/SubCategory/index.js [hunk 1]
```diff
@@ -136,7 +136,9 @@ const SubCategory = ({ categoryName, isFormDisabled, subCategoryName, actions, p
       </Box>
       {isModalOpen && (
         <ConditionsModal
-          headerBreadCrumbs={[categoryName, subCategoryName]}
+        headerBreadCrumbs={[
+          { id: categoryName, defaultMessage: categoryName },
+          { id: subCategoryName, defaultMessage: '' }]}
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/PluginsAndSettings/SubCategory/index.js:141]: thank you for the update, looks great!
could we just use `subCategoryName` as `defaultMessage` here @fhenri42? :)

**Review by ronronscelestes — COMMENTED** [review]
hi @fhenri42, thank you for the update 😊
there seems to be some lint issues, could you fix them please?
image

there's also a missing CLA signature that I can see:
image

after that we'll be ready to merge 🎉

**Review by ronronscelestes — COMMENTED** [review]
it looks great thank you 🎉
there's still an issue with the CLA though, the merging process is blocked, could you please have a look into it? we'll be then able to merge for the next release 🙂

image

**Comment by fhenri42** [comment]
Hello,
I am sorry, but I did not work enough on this project to be able to fix the coverage issue. I have some idea what I need to change, but some guidance are welcomed.

**Comment by ronronscelestes** [comment]
> Hello, I am sorry, but I did not work enough on this project to be able to fix the coverage issue. I have some idea what I need to change, but some guidance are welcomed.

hi! the coverage is not an issue :)
the issue is about signing the CLA (Contributor License Agreement)
as you can see in the screenshot (you can find this message in an automated comment posted by `strapi-cla` above) there is an unsigned commit which prevents me to merge the PR
image

by looking at the commits it seems that those commits are the one not signed:
image

let me know if you need any more help :)

**Comment by fhenri42** [comment]
ahah okay :D my bad !
But this user does not exist "Floren Henri", it's my GitLab account setup on my computer, how can I sign the CLI if no account is set up ?

**Comment by derrickmehaffy** [comment]
You will have to clean the entire commit history to change the author of the commits.

**Pull request closed without merging**